### ISSUE TITLE
forum

### ISSUE THREAD

**Issue by pygamegirl** [pygame/pygame#3389]
Hi

I'm new to pygame but I think this is for bugs here.
Unfortunately I can't find a forum for pygame.
Where can beginners ask questions about pygame or get help when they are stuck with some code?

Thank you

**Comment by robertgoodwin00** [comment]
As far as asking questions when stuck, Stack Overflow is the only place I know.
Unfortunately Pygame doesn't seem to have any central forum. (I've looked and looked) Nor even a place to share your finished game since the main site doesn't allow registrations. It's really unfortunate.

**Comment by blankRiot96** [comment]
There is an excellent <a href="https://reddit.com/r/pygame">subreddit</a> 
There is stackoverflow as mentioned

Some good non forum related options,
There is a pretty active <a href="https://discord.gg/KcJVXBaR7j">discord server</a> dedicated to pygame where many of the contributors discuss and many beginners receive help 
Large discord servers like the Python discord and coding den have places where you can ask for help as well.

**Comment by Starbuck5** [comment]
Yes, there is an active subreddit and an active discord, which are both places you could get help or show off games.

Going to go ahead and close this because I believe the question has been answered.

**Comment by pygamegirl** [comment]
Is there no interest in making a forum on pygame's website?
It's not that I don't like discord but a forum is nice to have different threads to group discussions or games.
On discord you talk a bit across all things that are said and the 'overview' after some time gets lost.
At least, that's how I feel it.

**Comment by Starbuck5** [comment]
Good suggestion, I added relevant links on that page.

I don't think pygame is going to do a forum on its website anytime soon, but I think these existing communities will help you out 🙂
